assets/js/vue/components/multi-search/ResultSet.vue

```diff
@@ -93,18 +91,22 @@ export default {
   data() {
     return {
       facetLabels: {
-        researchGroup: {
+        researchGroupInfo: {
           label: templateSwitch.getProperty('researchGroup'),
           queryParam: 'researchGroup',
         },
-        fundingOrg: {
+        fundingOrgInfo: {
           label: templateSwitch.getProperty('fundingOrg'),
           queryParam: 'fundingOrg',
         },
-        dataType: {
+        dataTypeInfo: {
           label: 'Type',
           queryParam: 'dataType',
         },
```

Comment:
I don't think they want this anymore, also it doesn't work anymore, because they are all datasets. If it would would, it should have only Dataset type in that agregation.